packages/components/src/Actions/ActionFile/__snapshots__/ActionFile.test.js.snap

```diff
@@ -41,6 +41,7 @@ exports[`ActionFile should display a Progress indicator if set 1`] = `
   >
     <withI18nextTranslation(CircularProgress)
       size="small"
+      t={[Function]}
```

Comment:
A reminder: talend/scripts i18next HOC returns the component, with a default props.t that is a mock of t function.
In this jest version, default props appear in the snapshots.